src/components/ProtonProducts/ProtonProducts.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import "../../pages/index.scss"
+
+function ProotonProducts(props) {
+  return (
+    <div className="ProtonProducts w50 relative onmobile-w100 " style={{ backgroundColor: props.color }}>
+      <img className="ProtonMailLogo mrauto mlauto " src={props.logo} />
+      <p className="ProtonMailText mrauto mlauto ">{props.text}</p>
+      <a className="ProtonMailButton">{props.button}</a>
+      <div className="ProtonMailPic">
+        <img src={props.picture} width="100%" />
+      </div>
+      <div className="WhiteDiv"></div>
```

Comment:
I guess this might be set up using a CSS gradient, something like:

```css
background-image: linear-gradient(to bottom, #657ee4 0%, #657ee4 75%, #fff 75% , #fff 100%);
```
(careful, white on white, open the image in another tab)

To test and adapt to get this 
![image](https://user-images.githubusercontent.com/2578321/91724464-a3e04c00-eb9d-11ea-988e-8e4a99781152.png) 
I've put the 75% value just to give an idea.